smf/internal-dns/manifest.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
```

Comment:
I assume this is following a pattern of other services.  Has @jclulow taken a look at this?